multiuser/keycloak/che-multiuser-keycloak-user-remover/src/test/java/org/eclipse/che/multiuser/keycloak/server/KeycloakUserRemoverTest.java

```diff
@@ -0,0 +1,81 @@
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.multiuser.keycloak.server;
+
+import static org.eclipse.che.multiuser.keycloak.shared.KeycloakConstants.REALM_SETTING;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.initMocks;
+import static org.testng.Assert.fail;
+
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.Map;
+import org.eclipse.che.api.core.NotFoundException;
+import org.eclipse.che.api.core.ServerException;
+import org.eclipse.che.api.core.rest.DefaultHttpJsonRequest;
+import org.eclipse.che.api.core.rest.HttpJsonRequestFactory;
+import org.mockito.Mock;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class KeycloakUserRemoverTest {
+
+  @Mock private KeycloakSettings keycloakSettings;
+  @Mock private OIDCInfo oidcInfo;
+  @Mock private HttpJsonRequestFactory requestFactory;
+  @Mock private KeycloakTokenRequester tokenRequester;
+  @Mock private DefaultHttpJsonRequest jsonRequest;
+
+  private KeycloakUserRemover keycloakUserRemover;
+
+  @BeforeMethod
+  public void setUp() throws ServerException {
+    initMocks(this);
+
+    Map<String, String> settingsMap = new HashMap<>();
+    settingsMap.put(REALM_SETTING, "realm");
+    when(keycloakSettings.get()).thenReturn(settingsMap);
+    when(oidcInfo.getAuthServerURL()).thenReturn("auth.server.url");
+    when(requestFactory.fromUrl(anyString())).thenReturn(jsonRequest);
+    when(tokenRequester.requestToken(anyString(), anyString(), anyString())).thenReturn("token");
+    when(jsonRequest.setAuthorizationHeader(anyString())).thenCallRealMethod();
+    when(jsonRequest.useDeleteMethod()).thenCallRealMethod();
+    when(jsonRequest.setMethod(anyString())).thenCallRealMethod();
+    keycloakUserRemover =
+        new KeycloakUserRemover(
+            true, "admin", "admin", keycloakSettings, tokenRequester, oidcInfo, requestFactory);
+  }
+
+  @Test
+  public void shouldCatchFileNotFoundException() throws Exception {
+    when(jsonRequest.request()).thenThrow(FileNotFoundException.class);
+    try {
+      keycloakUserRemover.removeUserFromKeycloak("123");
+    } catch (Exception e) {
+      fail("Should not have thrown an exception.");
+    }
+    verify(jsonRequest).request();
+  }
+
+  @Test
+  public void shouldCatchNotFoundException() throws Exception {
+    when(jsonRequest.request()).thenThrow(NotFoundException.class);
+    try {
+      keycloakUserRemover.removeUserFromKeycloak("123");
+    } catch (Exception e) {
+      fail("Should not have thrown an exception.");
```

Comment:
not necessary.

Reply:
I removed the message in my latest commits